src/main/java/com/minecolonies/colony/jobs/JobBuilder.java

```diff
@@ -119,13 +120,18 @@ public WorkOrderBuild getWorkOrder()
     }
 
     /**
-     * Set a Work Order for this Job
+     * Set a Work Order for this Job.
      *
      * @param order Work Order to associate with this job, or null
      */
     public void setWorkOrder(WorkOrderBuild order)
     {
-        workOrderId = (order != null) ? order.getID() : 0;
+        if (order == null)
```

Comment:
Can we please change this back, it was one nice statement, even and if () {} else would be better than the return.